### PR TITLE
Hmis 326 develop

### PR DESCRIPTION
**Pull request by mjotichmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMIS-326

### Change description ###
New endpoint that will retrieve Hearings and supports following parameters

- Hearing Id
- Hearing Date
- Hearing Type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by subrotomukherjee — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java [hunk 1]
```diff
@@ -0,0 +1,396 @@
+package uk.gov.hmcts.futurehearings.hmi.unit.testing.testsuites;
+
+import com.aventstack.extentreports.ExtentTest;
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.*;
```
Comment (pats-john) [inline src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java:6]: Remove Wildcards from imports and make it class specific please...

**Review by pats-john — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java (hunk 1 above)
Comment (git-sabs) [reply src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java]: Expanded the import statements as part of the commit below

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java [hunk 2]
```diff
@@ -16,7 +24,10 @@
 
 import static io.restassured.RestAssured.given;
 import static uk.gov.hmcts.futurehearings.hmi.unit.testing.util.ResponseVerifier.*;
```
Comment (pats-john) [inline src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java:26]: ResponseVerifier as well please....

**Review by pats-john — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java (hunk 2 above)
Comment (git-sabs) [reply src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java:26]: We are using all the methods in the responseverifier class

**Pull request merged**